ChocoPacker.UI/app/index.html

```diff
@@ -10,7 +10,62 @@
 <header class="app-header">
   <h1 class="logo">ChocoPacker</h1>
 </header>
```

Comment:
put it only with demo purpose, like a stub to show that content is stretchable and fills the full width of a row.